SmartDeviceLink/SDLPutFileResponse.h

```diff
@@ -15,10 +15,15 @@ NS_ASSUME_NONNULL_BEGIN
 @interface SDLPutFileResponse : SDLRPCResponse
 
 /**
- Provides the total local space available in SDL Core for the registered app. If the transfer has systemFile enabled, then the value will be set to 0 automatically.
+ *  Provides the total local space available in SDL Core for the registered app. If the transfer has systemFile enabled, then the value will be set to 0 automatically.
  */
 @property (strong, nonatomic) NSNumber<SDLInt> *spaceAvailable;
 
+/**
+ *  The success status of the data sent to SDL Core. 
+ */
+@property (nullable, strong, nonatomic) SDLResult resultCode;
```

Comment:
Why does this have an `SDLResult`? It seems to be mimicking the superclass `SDLResult`?

Reply:
Fixed. Removed the `resultCode` parameter as the superclass has the same parameter.